packages/governance/src/typeGuards.js

```diff
@@ -0,0 +1,103 @@
+// @ts-check
+
+import { M } from '@agoric/store';
+import { TimestampShape } from '../../SwingSet/src/vats/timer/typeGuards';
+
+export const ChoiceMethodShape = M.or('unranked', 'order');
+export const QuorumRuleShape = M.or('majority', 'no_quorum', 'all');
+export const ElectionTypeShape = M.or(
+  'param_change',
+  'election',
+  'survey',
+  'api_invocation',
+  'offer_filter',
+);
+
+export const TimerShape = M.handle('timer');
```

Comment:
For a remotable, the most that the pattern system can enforce is that it is a remotable. The string argument is only for diagnostic purposes. So the following does all the enforcement you have in mind, but by itself fails to provide the diagnostic info that you're expecting a handle.
```suggestion
export const TimerShape = M.remotable('timer');
```
you could just indicate that in the name manually
```suggestion
export const TimerShape = M.remotable('timerHandle');
```
or, if you think it's worth it, create an abstraction that just does the name mangling
```suggestion
const makeHandleShape = name => M.remotable(`${name}Handle`);
//...
export const TimerShape = makeHandleShape('timer');
```

Reply:
cool!  thanks.